kernel/thread.c

```diff
@@ -411,6 +412,9 @@ static tcb_t *thread_select(tcb_t *parent)
 	if (thr == NULL)
 		return NULL;
 
+	if ((thr = rr_select()) != NULL)
```

Comment:
Is like insmod/rmmod or config in `make config`? 

Thanks.

Also, Linux kernel scheduler is pluggable!? Ain't they choose when config?